projects/packages/videopress/src/client/block-editor/blocks/video/components/details-panel/index.native.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * WordPress dependencies
+ */
+import { PanelBody, BottomSheetTextControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import { VideoControlProps } from '../../types';
+/**
+ * Types
+ */
+import type React from 'react';
+
+/**
+ * React component that renders the details settings panel.
+ *
+ * @param {VideoControlProps} props - Component properties.
+ * @returns {React.ReactElement}	- Details panel component.
+ */
+export default function DetailsPanel( { attributes, setAttributes }: VideoControlProps ) {
+	const { title, description } = attributes;
+
+	return (
+		<PanelBody title={ __( 'Details', 'jetpack-videopress-pkg' ) }>
+			<BottomSheetTextControl
```

Comment:
I think this component is generally used for multiline content, however, in this case, the title seems a one-line value. I'm wondering if we could use a different component like [`TextControl`](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/text-control/README.md), WDYT?

Reply:
@SiobhyB @fluiddot hey this seems like a good idea to me. Happy with that change. Noting that it might mean some of the title is not visible if its long. But the user can still select within it to see the other characters. So it seems worth the benefit of not having to drill down 👍

Reply:
Thanks both! I've gone ahead to make that change in https://github.com/Automattic/jetpack/pull/29283/commits/80804d4940839637381361085297efd408d21370 and have re-requested your review @fluiddot :)